swatch-core/src/main/java/org/candlepin/subscriptions/logback/RhsmSplunkHecEventHeaderSerializer.java

```diff
@@ -0,0 +1,21 @@
+package org.candlepin.subscriptions.logback;
+
+import com.splunk.logging.EventHeaderSerializer;
+import com.splunk.logging.HttpEventCollectorEventInfo;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RhsmSplunkHecEventHeaderSerializer implements EventHeaderSerializer {
+  @Override
+  public Map<String, Object> serializeEventHeader(
+      HttpEventCollectorEventInfo eventInfo, Map<String, Object> metadata) {
+
+    var fields = (Map<String, Object>) metadata.getOrDefault("fields", new HashMap<>());
+
+    // TODO this better
+    fields.put("namespace", System.getenv("SPLUNKMETA_namespace"));
```

Comment:
I took a stab at trying to make this configurable via a properties bean or something, but I think it's invoked at an awkward part of the lifecycle.  This "works", but ideally I'd like this not to be so buried.